mmcv/runner/checkpoint.py

```diff
@@ -352,22 +352,20 @@ def load_from_ceph(filename, map_location=None, backend='petrel'):
     Returns:
         dict or OrderedDict: The loaded checkpoint.
     """
-    allowed_backends = ['ceph', 'petrel']
+    allowed_backends = ['ceph', 'petrel', 'aws']
```

Comment:
Maybe we can do it another way to determine if the backend is the right s3 backend.

Reply:
I modified it to judge through prefix s3.